tests/Maker/MakeMigrationTest.php

```diff
@@ -29,6 +29,12 @@ public function getTestDetails()
             // only requires doctrine/dbal. But we're testing with the ORM,
             // so let's install it
             ->addExtraDependencies('doctrine/orm:@stable')
+            /*
+             * @legacy doctrine/migrations does not support psr/log >=2
+             *         some symfony components use psr/log ^1 || ^2 || ^3
+             *         once doctrine/migrations supports psr log >= 2, this line can be removed.
+             */
+            ->addExtraDependencies('psr/log:^1.1.4')
```

Comment:
@weaverryan need to make sure im not missing something here - w/o this "fix" we get the following in CI and locally. See the description in the PR comments
```
1) Symfony\Bundle\MakerBundle\Tests\Maker\MakeMigrationTest::testExecute with data set "migration_with_changes" (Symfony\Bundle\MakerBundle\Test\MakerTestDetails Object (...))
Exception: Error running command: "composer require migrations doctrine/orm". Output: "". Error: "Using version ^3.1 for doctrine/doctrine-migrations-bundle
Using version ^2.9 for doctrine/orm
composer.json has been updated
Running composer update doctrine/doctrine-migrations-bundle doctrine/orm
Loading composer repositories with package information
Restricting packages listed in "symfony/symfony" to "5.4.*"
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - doctrine/doctrine-migrations-bundle[3.1.0, ..., 3.1.1] require doctrine/migrations ^3.1 -> satisfiable by doctrine/migrations[3.1.0, ..., 3.3.x-dev].
    - doctrine/doctrine-migrations-bundle[3.1.x-dev, ..., 3.2.x-dev] require doctrine/migrations ^3.2 -> satisfiable by doctrine/migrations[3.2.0, 3.2.x-dev, 3.3.x-dev].
    - doctrine/migrations[3.1.0, ..., 3.3.x-dev] require psr/log ^1.1.3 -> found psr/log[1.1.3, 1.1.4] but the package is fixed to 2.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires doctrine/doctrine-migrations-bundle ^3.1 -> satisfiable by doctrine/doctrine-migrations-bundle[3.1.0, 3.1.1, 3.1.x-dev, 3.2.x-dev].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

Installation failed, reverting composer.json and composer.lock to their original content.
"

/var/htdocs/src/Test/MakerTestProcess.php:51
/var/htdocs/src/Test/MakerTestEnvironment.php:154
/var/htdocs/src/Test/MakerTestCase.php:55
/var/htdocs/src/Test/MakerTestCase.php:33
```

Reply:
No, I think this is correct. We need to watch https://github.com/doctrine/migrations/pull/1184 and remove this when that's merged and there's a tag.

Reply:
Issue added: https://github.com/symfony/maker-bundle/issues/928